api/jobs.go

```diff
@@ -207,11 +210,13 @@ func (api *API) GetJobsHandler(w http.ResponseWriter, req *http.Request) {
 		return
 	}
 
+	// update links for json response
 	for i := range jobs.JobList {
 		jobs.JobList[i].Links.Self = fmt.Sprintf("%s/%s%s", host, v1, jobs.JobList[i].Links.Self)
 		jobs.JobList[i].Links.Tasks = fmt.Sprintf("%s/%s%s", host, v1, jobs.JobList[i].Links.Tasks)
 	}
 
+	// write response
```

Comment:
(suggestion): not sure if you need this comment, as the api is clear on its purpose.

Reply:
I wrote the comments so that we can understand the high level of what the handler is doing by just reading the comments alone. And then if needed, we can dig through the code